apps/condo/domains/organization/components/OrganizationSelect.tsx

```diff
@@ -32,28 +32,38 @@ function compareEmployees (lhs: OrganizationEmployeeType, rhs: OrganizationEmplo
 
 const DROPDOWN_OVERLAY_STYLES: CSSProperties = { maxWidth: 300, width: '100%' }
 
+const ORGANIZATION_EMPLOYEE_WHERE_QUERY = {
+    isRejected: false,
+    isBlocked: false,
+    organization: { type_not: HOLDING_TYPE },
+}
+
 export const InlineOrganizationSelect: React.FC = () => {
     const intl = useIntl()
     const AddOrganizationTitle = intl.formatMessage({ id: 'pages.organizations.CreateOrganizationButtonLabel' })
 
     const { breakpoints } = useLayoutContext()
     const textSize: TypographyTextProps['size'] = !breakpoints.TABLET_LARGE ? 'small' : 'medium'
 
-    const router = useRouter()
-
     const { user } = useAuth()
     const { link, selectLink, isLoading: organizationLoading } = useOrganization()
     const userId = get(user, 'id', null)
 
     const { objs: userEmployees, allDataLoaded: employeesLoaded } = OrganizationEmployee.useAllObjects({
         where: {
             user: { id: userId },
-            isRejected: false,
-            isBlocked: false,
             isAccepted: true,
-            organization: { type_not: HOLDING_TYPE },
+            ...ORGANIZATION_EMPLOYEE_WHERE_QUERY,
+        },
+    }, { skip: isNull(userId) })
+
+    const { count: hasInvites, loading: isInvitesLoading } = OrganizationEmployee.useCount({
+        where: {
+            user: { id: userId },
+            isAccepted: false,
+            ...ORGANIZATION_EMPLOYEE_WHERE_QUERY,
         },
-    })
+    }, { skip: isNull(userId) })
```

Comment:
Isn't skip is `Int`?

Reply:
Why not just `!userId` :)

Reply:
If I'm not mistaken, I used an apollo graphql client hook parameter. According to the documentation it has a boolean type
<img width="725" alt="image" src="https://github.com/open-condo-software/condo/assets/25844927/b5d5be48-aefe-46ab-933a-7f62adf6610c">

Reply:
> Why not just `!userId`? :)

Yup, it is possible